project/kediatr-core/src/main/kotlin/com/trendyol/kediatr/PipelineBehavior.kt

```diff
@@ -10,24 +10,7 @@ interface PipelineBehavior {
      * Process to invoke before handling any query, command or notification
      *
      * @param request the request to handle
+     * @param next the represents the CommandHandler handle function
      */
-    suspend fun <TRequest> preProcess(request: TRequest)
```

Comment:
Introducing a new alias for `next` in the pipeline makes it easier to access.

`typealias RequestHandlerDelegate<TRequest, TResponse> = suspend (TRequest) -> TResponse`

Reply:
Thanks for the typealias suggestion 🙏